ambry-network/src/main/java/com.github.ambry.network/Transmission.java

```diff
@@ -132,8 +132,7 @@ public SocketAddress getRemoteSocketAddress() {
   /**
    * Close the connection for the socket channel
    */
-  public abstract void close()
-      throws IOException;
```

Comment:
Any idea why `Transmission` is a public class? Doesn't seem like something that should be exposed outside of `Network`. (Not recommending making any changes to this class at this time).